tests/ebuild/test_eclass.py

```diff
@@ -64,13 +64,13 @@ def make_eclass(name, provides=None):
 # }
 # @CODE
 
-# @ECLASS-VARIABLE: _FOO_INTERNAL_ECLASS_VAR
+# @ECLASS_VARIABLE: _FOO_INTERNAL_ECLASS_VAR
 # @INTERNAL
 # @DEFAULT_UNSET
 # @DESCRIPTION:
 # Internal variable.
 
-# @ECLASS-VARIABLE: FOO_PUBLIC_ECLASS_VAR
+# @ECLASS_VARIABLE: FOO_PUBLIC_ECLASS_VAR
```

Comment:
How about keeping the old syntax in one of them for better test coverage? Either that or adding a dedicated test for the compat logic.

Reply:
I've added `FOO_ANOTHER_ECLASS_VAR` with the old syntax.